packages/typescript/ai-memory/tests/in-memory.test.ts

```
@@ -0,0 +1,4 @@
+import { runMemoryAdapterContract } from './contract'
+import { inMemoryMemoryAdapter } from '../src/adapters/in-memory'
```

ESLint

[error] `../src/adapters/in-memory` import should occur before import of `./contract`

(import/order)